docs/_docs/monitoring-metrics/system-views.adoc

```diff
@@ -601,76 +601,42 @@ Page lists organized in buckets. Each bucket group pages with about the same siz
-|REBALANCE_BATCH_SIZE | int | Size (in bytes) to be loaded within a single rebalance message
-|REBALANCE_BATCHES_PREFETCH_COUNT | int | Number of batches generated by supply node at rebalancing start
-|REBALANCE_DELAY | long |    Rebalance delay in milliseconds
-|REBALANCE_MODE | string |  Rebalance mode
-|REBALANCE_ORDER | int | Rebalance order
-|REBALANCE_THROTTLE | long |    Time in milliseconds to wait between rebalance messages to avoid overloading of CPU or network
-|REBALANCE_TIMEOUT | long |    Rebalance timeout in milliseconds
-|SQL_INDEX_MAX_INLINE_SIZE | int | Index inline size in bytes
-|SQL_ONHEAP_CACHE_MAX_SIZE | int | Maximum SQL on-heap cache. Measured in number of rows. When maximum size is reached oldest cached rows will be evicted.
-|SQL_SCHEMA | string |  Schema name
-|TOPOLOGY_VALIDATOR | string |  toString representation of topology validator
-|WRITE_BEHIND_BATCH_SIZE | int | Maximum batch size for write-behind cache store operations
-|WRITE_BEHIND_COALESCING | boolean | Write coalescing flag for write-behind cache store operations. Store operations (get or remove) with the same key are combined or coalesced to single, resulting operation to reduce pressure to underlying cache store
-|WRITE_BEHIND_FLUSH_FREQUENCY | long |    Frequency with which write-behind cache is flushed to the cache store in milliseconds
-|WRITE_BEHIND_FLUSH_SIZE | int | Maximum size of the write-behind cache. If cache size exceeds this value, all cached items are flushed to the cache store and write cache is cleared
-|WRITE_BEHIND_FLUSH_THREAD_COUNT | int | Number of threads that will perform cache flushing
-|WRITE_SYNCHRONIZATION_MODE | string |  Gets write synchronization mode
+=== CACHE_GROUP_PAGE_LISTS
+
+[{table_opts}]
+|===
+|Column | Data type |  Description
+|CACHE_GROUP_ID |  int| Cache group ID
+|PARTITION_ID |    int| Partition ID
+|NAME |    string|  Page list name
+|BUCKET_NUMBER |   int| Bucket number
+|BUCKET_SIZE | long  |  Count of pages in the bucket
+|STRIPES_COUNT |   int| Count of stripes used by this bucket. Stripes are used to avoid contention.
+|CACHED_PAGES_COUNT |  int| Count of pages in onheap page list cache for this bucket
+|===
+
+=== DATA_REGION_PAGE_LISTS
+
+[{table_opts}]
+|===
+|Column | Data type |  Description
+|NAME |    string|  Page list name
+|BUCKET_NUMBER |   int| Bucket number
+|BUCKET_SIZE | long  |  Count of pages in the bucket
+|STRIPES_COUNT |   int| Count of stripes used by this bucket. Stripes are used to avoid contention.
+|CACHED_PAGES_COUNT |  int| Count of pages in onheap page list cache for this bucket
+|===
+
+== PARTITION_STATES
 
-////
+This view exposes information about distribution of cache group partitions across cluster nodes.
+
+[{table_opts}]
+|===
+|Column | Data type |  Description
+|CACHE_GROUP_ID |  int| Cache group ID
+|PARTITION_ID |    int| Partition ID
+|NODE_ID | UUID | Node ID
+|STATE | string | Partition state
+|IS_PRIMARY | boolean  | Primary partition flag
```

Comment:
@alex-plekhanov could you add and elaborate on all the possible states? Sorry, I came across this attribute description only after merging your changes. Feel free to commit the changes to the branch directly, skipping the pull-request phase.

Reply:
Done